modules/url.py

```diff
@@ -64,13 +67,13 @@
 except:
     print 'WARNING: No simple_channels.txt found'
 
-
 url_finder = re.compile(r'(?iu)(%s?(http|https|ftp)(://\S+\.?\S+/?\S+?))' %
                         (EXCLUSION_CHAR))
 r_entity = re.compile(r'&[A-Za-z0-9#]+;')
 INVALID_WEBSITE = 0x01
 HTML_ENTITIES = { 'apos': "'" }
 
+yt_catch = re.compile('http[s]*:\/\/[w\.]*(youtube.com/watch\S*v=|youtu.be/)([\w-]+)')
```

Comment:
The use of wildcards here is kind of extreme. Anticipate http or https, and only one subdomain I'd imagine, like:

```
http[s]?:\/\/[w\.]?(...)
```